packages/core/src/result-transformers.ts

```diff
@@ -52,7 +52,8 @@ class ResultTransformers {
    * const { keys, records, summary } = await driver.executeQuery('CREATE (p:Person{ name: $name }) RETURN p', { name: 'Person1'})
    *
    * @returns {ResultTransformer<EagerResult<Entries>>} The result transformer
-   * @alias {@link ResultTransformers#eager}
+   * @deprecated This is deprecated, use {@link ResultTransformers#eager} instead
+   * @since 6.0.0
```

Comment:
Reading https://jsdoc.app/tags-since and https://jsdoc.app/tags-deprecated it seems that `@since` is not for marking since when something is deprecated. Consider including that information after the `@deprecated` tag instead.

Same for `#mappedResultTransformer`, won't comment again.